tests/inte_test/test_drmmtks.py

```diff
@@ -0,0 +1,78 @@
+import os
+import shutil
+
+import numpy as np
+import pytest
+
+import matchzoo as mz
+
+
+@pytest.fixture(scope='module')
+def train_data():
```

Comment:
These `train_data`s in integration tests are kinda redundant. I will refactor those in the future.